plugin/storage/es/esCleaner.py

```diff
@@ -1,11 +1,13 @@
 import elasticsearch
 import curator
+import logging
 import sys
+import os
 
 
 def main():
     if len(sys.argv) == 1:
-        print('USAGE: %s NUM_OF_DAYS HOSTNAME[:PORT] ...' % sys.argv[0])
+        print('USAGE: [TIMEOUT=(default 120)] %s NUM_OF_DAYS HOSTNAME[:PORT] ...' % sys.argv[0])
```

Comment:
this happens in `./es_indices_clean.sh`
technically i don't need this piece of code in the python script, but should i add an explanation to both anyways?